configs/app.dev.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I'm actually going to change that so it's easier as per @cindyli's suggestion in the channel yesterday. But I figured I might as well get you what you need for HST first. :)